bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -87,21 +93,23 @@ def instructor_flow(request, collection_slug=None, group_slug=''):
     return redirect(
         reverse(
             'module:collection-detail',
-            kwargs={'pk': Collection.objects.get(slug=collection_slug).id}
+            kwargs={'slug': collection_order.collection.slug}
         )
     )
 
 
 def create_sequence_item(request, sequence, start_activity, tool_provider, lti_consumer):
-    """Create and return sequence item."""
+    """
+    Create and return sequence item.
+    """
     # NOTE(wowkalucky): empty Collection validation
     cache.set(str(sequence.id), request.session['Lti_session'])
 
     # NOTE(wowkalucky): save outcome service parameters when Sequence is created:
     if tool_provider.is_outcome_service():
         outcomes, __ = OutcomeService.objects.get_or_create(
             lis_outcome_service_url=tool_provider.launch_params.get('lis_outcome_service_url'),
-            lms_lti_connection=lti_consumer
```

Comment:
Please rename lti_consumer into lti_lms and lti_provider into lti_content_source

Reply:
I renamed these models:
-  lti_consumer -> lti_content_sources
-  lti_provider -> lti_lms_platforms